Formula/tcptraceroute.rb

```diff
@@ -29,4 +41,9 @@ def caveats; <<-EOS.undent
     You should be certain that you trust any software you grant root privileges.
     EOS
   end
+
+  test do
+    output = shell_output("#{bin}/tcptraceroute --help 2>&1", 1)
```

Comment:
Is there a safe way to invoke sudo in the test sandbox? tcptraceroute requires sudo to actually conduct a trace.